src/blog/2018-9-26/csharp-and-scala-support.pug

```diff
@@ -0,0 +1,57 @@
+        * [Get started with Stryker4s](/stryker4s/quickstart)
+        * [Get started with Stryker](/stryker/quickstart) (for JavaScript / TypeScript)
+
+        ## New platforms
+
+        As you might know, both C# and Scala run on entirely different platforms compared to Stryker (which runs on NodeJS) does. This is the main reason we decided to implement them as
+        separate frameworks. [Stryker.NET](/stryker-net) is written in C# and runs as a .NET Core application, while [Stryker4s](/stryker4s) is written in Scala and runs on the JVM (Java Virtual Machine).
+        As of now, a version of Stryker.NET can be found on nuget.org, the package manager for the .NET platform. In due time, Stryker4s can be found on Maven Central.
+
+        Stryker.NET is the result of the internship of [Richard](https://github.com/richardwerkman)  
+        Stryker4s is the result of the internship of [Hugo](https://github.com/hugo-vrijswijk).
+
+        Thanks to them both for their hard work and continued support.
+
+        ## One design mentality
+        
+        Although all 3 versions of Stryker are implemented on different platforms, the design goals are the same. It should be _easy to use_ and
+        _fast to run_. Without compromises. 
+        
+        For Stryker JavaScript we're mutating _source code_ (as explained in our 
+        [road to Stryker 1.0](/blog/2017-07-14/road-to-stryker-1-0) blog post), instead of transpiled/minified JavaScript code. 
+        The main advantage is that it makes sure we only mutate _your actual code_, instead of 3rd party library or minified code.
+        
+        We wanted to offer the same quality to the .NET and Scala counterparts, but they have one additional problem: compiling code takes a long time.
+        It would dramatically reduce the performance of the frameworks. This is why we chose to implement a new mutation testing technique we call _Mutation Switching_.
+        With this technique, we compile only once. This results in a drastic performance gain, without compromising the quality of the mutations. 
+        It really deserves it's own blog post, so more on that later.
+        
+        ## Collaboration
+
+        Though the 3 frameworks have their own code base, there are actually a lot of points on which we're planning to collaborate.
+
+        * **One vocabulary**\
+          Terms like _mutant_, _survived_ and _mutator_ will mean the same across versions. We will even share the names
+          of the mutators, see [the supported mutators in our handbook](https://github.com/stryker-mutator/stryker-handbook/blob/master/mutator-types.md#supported-mutators)
+          for a comprehensive list of supported mutators.
+        * **One html reporter**\
+          We're redesigning our html report as [HTML 5 web components](https://www.webcomponents.org/). Interested to help? Please let us know! [github repository here](https://github.com/stryker-mutator/mutation-testing-elements).
```

Comment:
Maybe here could be added that the reporter will also be available for other "non-stryker" frameworks?